app/forms/waste_carriers_engine/cease_or_revoke_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  class CeaseOrRevokeForm < BaseForm
+    def self.can_navigate_flexibly?
+      false
+    end
+
+    delegate :metaData, to: :transient_registration
+    delegate :status, :revoked_reason, to: :metaData, allow_nil: true
+    delegate :contact_address, :company_name, :registration_type, :tier, to: :transient_registration
+
+    validate :validate_status
+    validate :validate_revoked_reason
+
+    private
+
+    def validate_status
+      return true if %w[INACTIVE REVOKED].include?(status)
```

Comment:
Just double-checking this. For example, I note `EXPIRED` is not listed.

As only an `ACTIVE` registration can be ceased or revoked, would it be easier to just check that `status == "ACTIVE"`?

Reply:
🤔 This validation is only for the form's params. The validation on the status to be active in order for the flow to work is in the permissions check. Why should this deal with expired?

Reply:
🤦‍♂